cpu/cc26x2_cc13x2/include/cc26x2_cc13x2_rf_internal.h

```diff
@@ -0,0 +1,395 @@
+/*
+ * Copyright (C) 2020 Locha Inc
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup         cpu_cc26x2_cc13x2
+ * @{
+ *
+ * @file
+ * @brief           CC26x2/CC13x2 IEEE 802.15.4g Sub-1-GHz driver internal
+ *                  functions.
+ *
+ * @author          Jean Pierre Dudey <jeandudey@hotmail.com>
+ */
+
+#ifndef CC26X2_CC13X2_RF_INTERNAL_H
+#define CC26X2_CC13X2_RF_INTERNAL_H
+
+#include <stdbool.h>
+
+#include "net/eui64.h"
+#include "iolist.h"
+#include "byteorder.h"
+#include "macros/units.h"
+
+#include "cc26xx_cc13xx_rfc_common_cmd.h"
+#include "cc26xx_cc13xx_rfc_prop_cmd.h"
+
+#include "cc26xx_cc13xx_rfc_mailbox.h"
+#include "cc26xx_cc13xx_rfc_prop_mailbox.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   Number of receive buffers
+ */
+#ifndef CONFIG_CC26x2_CC13X2_RF_RX_BUF_NUMOF
+#define CONFIG_CC26x2_CC13X2_RF_RX_BUF_NUMOF (4)
+#endif
+
+/* XXX: shouldn't this reside on net/ieee802154.h */
+/**
+ * @name    SUN PHY frequency band definitions
+ *
+ * These are the frequency band identifiers for the SUN PHY.
+ *
+ * @see IEEE Std 802.154.g-2012, Table 68f
+ * @{
+ */
+#define IEEE802154_SUN_PHY_FREQ_169_MHZ     0   /**< 169.400-169.475 (Europe) */
+#define IEEE802154_SUN_PHY_FREQ_450_MHZ     1   /**< 450-470 (US FCC Part 22/90) */
+#define IEEE802154_SUN_PHY_FREQ_470_MHZ     2   /**< 470-510 (China) */
+#define IEEE802154_SUN_PHY_FREQ_780_MHZ     3   /**< 779-787 (China) */
+#define IEEE802154_SUN_PHY_FREQ_863_MHZ     4   /**< 863-870 (Europe) */
+#define IEEE802154_SUN_PHY_FREQ_896_MHZ     5   /**< 896-901 (US FCC Part 90) */
+#define IEEE802154_SUN_PHY_FREQ_901_MHZ     6   /**< 901-902 (US FCC Part 24) */
+#define IEEE802154_SUN_PHY_FREQ_915_MHZ     7   /**< 902-928 (US) */
+#define IEEE802154_SUN_PHY_FREQ_917_MHZ     8   /**< 917-923.5 (Korea) */
+#define IEEE802154_SUN_PHY_FREQ_920_MHZ     9   /**< 920-928 (Japan) */
+#define IEEE802154_SUN_PHY_FREQ_928_MHZ     10  /**< 928-960 (US, non-contiguous) */
+#define IEEE802154_SUN_PHY_FREQ_950_MHZ     11  /**< 950-958 (Japan) */
+#define IEEE802154_SUN_PHY_FREQ_1427_MHZ    12  /**< 1427-1518 (US and Canada, non-contiguous) */
+#define IEEE802154_SUN_PHY_FREQ_2450_MHZ    13  /**< 2400-2483.5 */
```

Comment:
This could be moved to ieee802154 headers. Might be useful some day there outside of this driver if we manage to develop the descriptors present in the standard.

Don't know if @jia200x would agree on this